pudl_scrapers/spiders/eipinfrastructure.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+import scrapy
+from scrapy.http import Request
+
+from pudl_scrapers import items
+from pudl_scrapers.helpers import new_output_dir
+
+
+class EipInfrastructureSpider(scrapy.Spider):
+    name = 'eipinfrastructure'
+    allowed_domains = ['environmentalintegrity.org']
+
+
+    def start_requests(self):
+        """Finalize setup and yield the initializing request"""
+        # Spider settings are not available during __init__, so finalizing here
+        settings_output_dir = Path(self.settings.get("OUTPUT_DIR"))
+        output_root = settings_output_dir / "eipinfrastructure"
+        self.output_dir = new_output_dir(output_root)
+
+        yield Request("https://environmentalintegrity.org/download/eip-emissions-increase-database/?wpdmdl=13864&refresh=619e7743265e61637775171")
```

Comment:
I suspect that this URL is tied to exactly one particular file (the currently available data). If that's true, then so long as that file remains on their server, it'll appear that the data has never been updated when we re-run the scrapers.

If possible, instead it might be preferable to parse the link out of [the download page](https://environmentalintegrity.org/download/eip-emissions-increase-database/) based on whatever element of the document tree it's found in. Then when we re-run the scraper we would pull the new version if it's been updated, and hopefully be able to extract the date associated with the update from the filename.